cli/cmd/offsets.go

```diff
@@ -0,0 +1,235 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+
+	"github.com/spf13/cobra"
+
+	appsv1 "k8s.io/api/apps/v1"
+	v1 "k8s.io/api/core/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/odigos-io/odigos/api/k8sconsts"
+	cmdcontext "github.com/odigos-io/odigos/cli/pkg/cmd_context"
+	"github.com/odigos-io/odigos/cli/pkg/kube"
+	"github.com/odigos-io/odigos/common/consts"
+)
+
+var (
+	offsetFileMountPath = "/offsets"
+	useDefault          bool
+)
+
+var offsetsCmd = &cobra.Command{
+	Use:   "update-offsets",
```

Comment:
WDYT about adding it as subcommand to `odigos pro` so it's clear this feature is not expected to work for community. this will also keep the number of top level cli commands from growing.

Reply:
https://github.com/odigos-io/odigos/blob/8ca0480a57e4152a961d162bae095f217804b081/cli/cmd/pro.go#L22

Reply:
sgtm!